dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -70,6 +70,11 @@
   #plusPreloadAssetList
     %i.fa.fa-plus-circle
 
+.field
+  = f.label :default_song, 'Dance Lab Only - Default Song'
+  = f.select :default_song, options_for_select(@level.class.hoc_songs, @level.default_song)
```

Comment:
better to wrap this `.field` with `- if @level.is_a? Dancelab`.

the next time we add a setting specific to Dancelab, we should really extract a new _dancelab.html.haml partial, and then refer to it like this:  https://github.com/code-dot-org/code-dot-org/blob/66deb81a3be1ff69bb46948ee2c950aa436b50b0/dashboard/app/views/levels/_form.html.haml#L43